lib/options-manager.js

```diff
@@ -309,6 +309,13 @@ const buildESLintConfig = options => config => {
 		};
 	}
 
+	// ESLint does not expose a `fixDryRun` option in the programmatic API.
+	// Instead of using `fixDryRun` itself, the option is internally translated to `fix` option in ESLint's CLI module.
+	// https://github.com/eslint/eslint/blob/901ce0f1e32ea1e9e10ce4d8b37c0d750007a3c5/lib/cli.js#L96
```

Comment:
This does not make it clear how it then manages to be a "dry-run". The important part is missing.